cli/pcluster/cli.py

```diff
@@ -56,7 +58,8 @@ def update(args):
 
 
 def version(args):
```

Comment:
You can remove the `args` parameter from here too

Reply:
It's required by the way we call `argparse`, each function is called via `args.func(args)`. Removing it results in:

```bash
Unexpected error of type TypeError: version() takes no arguments (1 given)
```

I want to keep the invocation of all cli functions the same (I know command is different but that's for a good reason) hence leave the extra `args` parameter.